Cannot remove callbacks due to foreign key constraint

When trying to remove a callback URL, I get the following error:

```
Caused by: org.postgresql.util.PSQLException: ERROR: update or delete on table "pa_application_callback" violates foreign key constraint "pa_application_callback_id_fk" on table "pa_application_callback_event"
  Detail: Key (id)=(1c3ba2b2-f279-4265-87c9-c7ca8d22fabb) is still referenced from table "pa_application_callback_event".
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2733) ~[postgresql-42.7.4.jar:42.7.4]
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2420) ~[postgresql-42.7.4.jar:42.7.4]
	... 157 common frames omitted
```

The REST API method is `/rest/v3/application/callback/remove`.

--- a/powerauth-java-server/src/test/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/CallbackUrlBehaviorTest.java
+++ b/powerauth-java-server/src/test/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/CallbackUrlBehaviorTest.java
@@ -34,6 +34,7 @@ import io.getlime.security.powerauth.app.server.service.callbacks.CallbackUrlEve
 import io.getlime.security.powerauth.app.server.service.callbacks.model.CallbackUrlConvertor;
 import io.getlime.security.powerauth.app.server.service.exceptions.GenericServiceException;
 import io.getlime.security.powerauth.app.server.service.model.ServiceError;
+import io.getlime.security.powerauth.app.server.task.CleaningTask;
 import jakarta.persistence.EntityManager;
 import jakarta.transaction.Transactional;
 import org.junit.jupiter.api.Test;
@@ -72,6 +73,12 @@ class CallbackUrlBehaviorTest {
 
     @MockBean
     private CallbackUrlEventService callbackUrlEventService;
+
+    /**
+     * Mock CleaningTask to avoid running scheduled job when mocking CallbackUrlEventService
+     */
+    @MockBean
+    private CleaningTask cleaningTask;
 
     @Test
     void testCreateCallbackUrl() throws Exception {

